### PR TITLE
Add third-party-bots with caffe2-operator access

### PR DESCRIPTION
**Pull request by PatrickXYS**
Since we have a blocking sync issue as described #361 ,

this time we add one repo into the list, and meanwhile check the log to see how it goes.

This should give us more information to debug further.

/cc @jlewi

### CODE REVIEW

**Comment by Jeffwan** [comment]
This is just for test purpose, right?

**Comment by PatrickXYS** [comment]
@Jeffwan  Yes this is only for experimenting

File: github-orgs/kubeflow/org.yaml [hunk 1]
```diff
@@ -1011,6 +1011,7 @@ orgs:
             - aws-kf-ci-bot
             privacy: closed
             repos:
+              caff2-operator: write
               katib: write
```
Comment (jlewi) [inline github-orgs/kubeflow/org.yaml:1015]: Do you want to remove the other repos? Since it appears to be choking on the other repos?

**Comment by PatrickXYS** [comment]
Sounds good, let me start with tf-operator.

**Comment by PatrickXYS** [comment]
@jlewi  We can started with katib first, let's see how logs look like

Can you lgtm + approve?

**Pull request merged**